R/kinfitr_telemetry.R

```diff
@@ -0,0 +1,42 @@
+# set url_base to http://openneuropet.org for deployment
+url_base <- "http://54.144.240.214"
```

Comment:
```suggestion
url_base <- "http://openneuropet.org"
```

Apply this suggestion before merge!